tests/host_tools/network.py

```diff
@@ -60,8 +60,7 @@ def remote_path(self, path):
 
     def _scp(self, path1, path2, options):
         """Copy files to/from the VM using scp."""
-        ecode, _, stderr = self._exec(["scp", *options, path1, path2])
-        assert ecode == 0, stderr
+        self._exec(["scp", *options, path1, path2], check_output=True)
```

Comment:
use `check_output` here

Reply:
We can't, because `checck_output` delegates to `run`, which prepends the `ssh -i ... root@192.168.0.2` stuff, which we do not want for `scp`

Reply:
Ah right!